crates/api_crud/src/community/delete.rs

```diff
@@ -48,16 +48,14 @@ impl PerformCrud for DeleteCommunity {
     .map_err(|_| ApiError::err("couldnt_update_community"))?;
 
     // Send apub messages
-    if deleted {
-      updated_community
-        .blank_out_deleted_or_removed_info()
```

Comment:
What about all these blanking out deleted or removed info?

Reply:
Its not necessary because we are only sending the apub ID of a deleted/removed object.